db/postgres/schema.sql

```diff
@@ -1,52 +1,39 @@
 DROP DATABASE IF EXISTS sdc;
 
 CREATE DATABASE sdc;
+\c sdc;
 
--- \conect sdc;
+SET maintenance_work_mem to '16GB';
```

Comment:
Will this work on an EC2 instance?

Reply:
probably not. And i dont really need that there. It doesnt speed things up that much.